Dockerfile.rocky8

```diff
@@ -449,6 +449,7 @@ RUN if [ -n "${SMTP_SERVER}" ]; then \
       echo "# Simple SMTP setup to route all local mail to host SMTP" > /etc/esmtprc && \
       echo "hostname ${SMTP_SERVER}:${SMTP_PORT}" >> /etc/esmtprc && \
       echo "qualifydomain ${DOMAIN}" >> /etc/esmtprc ; \
```

Comment:
aren't you missing a replace of the ';' on the previous line with '&&' across all of the Dockerfiles to ensure that all echos of this particular RUN is executed successfully